test/EventHandler/main.cpp

```diff
@@ -342,32 +352,31 @@ static void test_event_assignment_and_swap() {
     call_event("e1", e1);
     call_event("e2", e2);
     call_event("e3", e3);
+    }
+    TEST_ASSERT_TRUE(MyArg::instcount == 0);
```

Comment:
Might be worth using a [`TEST_ASSERT_EQUAL(...)`](https://github.com/ARMmbed/unity/blob/master/README.md#equality) macro, which will print out the values if a failure occurs.


With some regex magic, you might be able to do this in one find and replace, so save some work (something like `TEST_ASSERT_TRUE(.*?==.*?);` to find similar things).
But not strictly speaking necessary.